.github/workflows/ci.yml

```diff
@@ -8,36 +8,36 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Use Node.js 14.x
+      - name: Use Node.js 14.17
```

Comment:
I think these workflows should end up resolving to 14.17 so should be good to leave it as `14.x`

Reply:
If I'm just leaving everything at 14.x, what exactly am I changing here?

Reply:
I think big thing is the `.nvrmc` change and then making sure both the repo runs/compiles on 14.17 and that the netlify previews work as expected 👀  (I think they use the .nvmrc file to pick up the version)

Totally get if it feels superfluous though! Don't sweat it if it doesn't feel like something you're interested in 👍